extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -868,6 +870,7 @@ export class AmpA4A extends AMP.BaseElement {
     dev().assert(this.uiHandler);
     this.uiHandler.setDisplayState(AdDisplayState.LOADING);
     this.uiHandler.setDisplayState(AdDisplayState.LOADED_NO_CONTENT);
+    this.collapse_ = true;
```

Comment:
Need to reset `collapse_` in `unlayoutCallback`.  Also, since it indicates state, should probably name it something like `isCollapsed_`.